src/Symfony/Bundle/FrameworkBundle/Resources/config/validator.xml

```diff
@@ -47,7 +48,6 @@
                     </call>
                 </service>
             </argument>
-            <deprecated>The "%service_id%" service is deprecated since Symfony 2.8 and will be removed in 3.0.</deprecated>
```

Comment:
Nevermind, this is *not* in 3.0 - my mistake. But for the purposes of getting this merged in a timely manner, I still think it's best to add it back and then decide to deprecate it in 3.0 (for 4.0 removal) - i.e. keep the PR as it's written.